src/ch/imedias/rsccfx/view/RsccShowTokenView.java

```diff
@@ -15,15 +14,9 @@
 import javafx.scene.text.Font;
 import javafx.scene.text.Text;
 
-
-/**
- * Class ShowTokenView
- * Created by Simon on 30.11.16.
- * Capsulated Class to test easy.
- */
-public class ShowTokenView extends BorderPane {
+public class RsccShowTokenView extends BorderPane {
   private Rscc model;
-  BorderPane pane;
+  BorderPane pane; // FIXME: rename all fields to comply with the SAD and google java code guidelines
```

Comment:
@Kennox rename all fields to comply with the SAD and google java code guidelines